pkg/virt-config/feature-gates.go

```diff
@@ -81,8 +81,8 @@ const (
 	//
 	// CommonInstancetypesDeploymentGate enables the deployment of common-instancetypes by virt-operator
 	CommonInstancetypesDeploymentGate = "CommonInstancetypesDeploymentGate"
-	// CPUManagerPolicyBetaOptionsGate allows conforming VM to static CPU-Manager Policies.
-	CPUManagerPolicyBetaOptionsGate = "CPUManagerPolicyBetaOptions"
+	// AlignCPUsGate allows emulator thread to assign two extra CPUs if needed to complete even parity.
```

Comment:
By the end of this PR, this FG gates the mutating web-hooks behavior to automatically add annotation on VMIs with IsolatedEmulatorThread.
Please consider aligning the comment to better explain the meaning of the FG.

Reply:
well what is said is still true, it doesn't matter that it's doing it on the kubevirt CR, and not the VMI.